index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Mobile Version</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div class="header-welcome">
+      <a href="/">
+        <img class="some-image" src="images/Welcome.svg" alt="Welcome" />
+      </a>
+    </div>
+
+    <nav>
+      <div class="menu-container">
+        <img class="some-image" src="images/icon-menu.svg" alt="Menu Icon">
+      </div>
+    </nav>
+  </header>
```

Comment:
- [ ] Kindly add some left right and top margins to this header section

reference |
------------ |
![image](https://user-images.githubusercontent.com/67957533/133605695-fccbc6e0-6c47-4b7c-9870-51780e5bd79a.png) |